full-stack-tests/imodelconnection-without-certa/src/Settings.ts

```diff
@@ -52,20 +52,20 @@ export class Settings {
   public iModel: IModelData = {} as IModelData;
 
   constructor() {
-    const envFile = path.join(__dirname, "..", "..", ".env");
-    if (!fs.existsSync(envFile))
-      return;
-
+    const path = require("path"); // eslint-disable-line @typescript-eslint/no-var-requires
     const dotenv = require("dotenv"); // eslint-disable-line @typescript-eslint/no-var-requires
     const dotenvExpand = require("dotenv-expand"); // eslint-disable-line @typescript-eslint/no-var-requires
-    const envResult = dotenv.config({ path: envFile });
-    if (envResult.error) {
-      throw envResult.error;
-    }
 
-    dotenvExpand(envResult);
+    // First check in process.cwd() for the config
+    let result = dotenv.config();
```

Comment:
@wgoehrig does this look right to you?

Reply:
@pmconne The current way I am loading the env into the test doesn't work for the integration test. It still currently doesn't, I'm not quite sure how to get the env file while in the pipeline.

Reply:
I left a few comments related to this... @AidanTran, you may want to check out https://github.com/iTwin/itwinjs-core/blob/master/full-stack-tests/ecschema-rpc-interface/src/backend/BackendInit.ts#L11 as an example of how we typically load a .env file...

Also, just to clarify, in our pipelines we don't actually use a .env file - the environment variables are set directly by the pipeline (usually from an Azure DevOps variable group)